docs/docs/config/battery.md

```diff
@@ -22,6 +22,12 @@ While `CONFIG_ZMK_BATTERY_REPORTING` is disabled by default it is implied by `CO
 
 :::
 
+:::note BLE reporting on MacOS
+
+On MacOS the BLE battery reporting packets can cause the computer to wakeup from sleep, the battery service can be disabled indepently of battery monitoring (allowing the battery level to be displayed on a screen or through a custom lighting implementation) by setting `CONFIG_BT_BAS=n`
```

Comment:
```suggestion
On macOS the BLE battery reporting packets can cause the computer to wakeup from sleep. To prevent this, the battery *reporting* service can be disabled by setting `CONFIG_BT_BAS=n`. This setting is independent of battery *monitoring*, for instance the battery level could still be indicated on a display.
```
Omitting the custom lighting part because it currently isn't a feature in ZMK, to prevent potential confusion.